### PR TITLE
Spanish translation update

### PR DESCRIPTION
**Pull request by EdgarLux**
Polishing the translation.

### CODE REVIEW

**Comment by TurboGit** [comment]
A bit too late, this will be for 3.0.1 now.

**Comment by EdgarLux** [comment]
> A bit too late, this will be for 3.0.1 now.

What a pity. It's ok, I understand. I wish I'd finished sooner. Thanks you.

File: po/es.po [hunk 1]
```diff
@@ -10529,35 +10530,35 @@ msgstr "reconstrucción de luces"
...
 msgstr ""
-"comprimir el efecto en luces/sombras y\n"
-"preservar tonos medios"
+"Comprimir el efecto en luces/sombras\n"
+"preservando tonos medios"
 
 #: ../src/iop/shadhi.c:872
 msgid "adjust saturation of shadows"
-msgstr "ajuste de la saturación de sombras"
+msgstr "Adecuar la saturación de sombras"
```
Comment (rgo) [inline po/es.po]: I would rather say "Ajustar la saturación de sombras", _adecuar_ is not a common verb.

File: po/es.po [hunk 2]
```diff
@@ -10529,35 +10530,35 @@ msgstr "reconstrucción de luces"
...
 
 #: ../src/iop/shadhi.c:873
 msgid "adjust saturation of highlights"
-msgstr "ajuste de la saturación de luces"
+msgstr "Adecuar la saturación de luces"
```
Comment (rgo) [inline po/es.po]: Same as before "Ajustar" is widely used.

File: po/es.po [hunk 3]
```diff
@@ -11128,10 +11129,11 @@ msgid ""
 "contrast\n"
 "'averaged guided filter' is a geometric mean of both methods"
 msgstr ""
-"'no' afecta el contraste global y local (seguro si solo agrega contraste)\n"
-"'filtro guiado' solo afecta el contraste global tratando de preservar el "
+"No: afecta el contraste global y local (es seguro si solo se añade "
+"contraste)\n"
+"Filtro guiado: solo afecta el contraste global tratando de preservar el "
```
Comment (rgo) [inline po/es.po]: Options were quoted in original sentences, why the change to colon as it was a definition?
I think, that it will remain as quoted word followed with an small explanation.

File: po/es.po [hunk 4]
```diff
@@ -11128,10 +11129,11 @@ msgid ""
...
 "contraste local\n"
-"'filtro guiado promediado' es una media geométrica de ambos métodos"
+"Filtro guiado promediado: es una media geométrica de ambos métodos"
```
Comment (rgo) [inline po/es.po]: Same comment about quotes and colon as before

File: po/es.po [hunk 5]
```diff
@@ -11271,23 +11273,23 @@ msgstr "desviación de tonos medios"
 
 #: ../src/iop/velvia.c:392
 msgid "the strength of saturation boost"
-msgstr "fuerza del incremento de saturación"
+msgstr "Intensidad del realce de la saturación"
```
Comment (rgo) [inline po/es.po]: Totally subjective opinion: I don't really like the "realce" word, it's kinda over-elaborate.

File: po/es.po [hunk 6]
```diff
@@ -11333,15 +11335,15 @@ msgstr "proporción automática"
...
 
 #: ../src/iop/vignette.c:1154
 msgid "16-bit output"
-msgstr "salida 16-bit"
+msgstr "salida de 16 bits"
 
 #: ../src/iop/vignette.c:1159
 msgid "fall-off strength"
-msgstr "fuerza del difuminado"
+msgstr "graduación del difuminado"
```
Comment (rgo) [inline po/es.po]: I don't think "graduación" is wrong but I consider simpler "fuerza".
_fall-off_ can be translated as _atenuación_ but I don't think it worth the time.

Just as a curiosity, adobe translates it as _desvanecimiento_  , I don't like it.

File: po/es.po [hunk 7]
```diff
@@ -11361,27 +11363,27 @@ msgstr "proporción anchura/altura"
 
 #: ../src/iop/vignette.c:1185
 msgid "the radii scale of vignette for start of fall-off"
-msgstr "radio de la escala de viñeteado del principio del difuminado"
+msgstr "Dimensión del radio donde inicia el difuminado del efecto viñeta"
```
Comment (rgo) [inline po/es.po]: A minor possible change  "Dimensión del radio donde **empieza** el difuminado del efecto viñeta"

File: po/es.po [hunk 8]
```diff
@@ -11398,15 +11400,15 @@ msgstr ""
 #: ../src/iop/vignette.c:1193
 msgid "enable to have the ratio automatically follow the image size"
 msgstr ""
-"activar para que la proporción siga el tamaño de imagen automáticamente"
+"Activar para que la proporción siga automáticamente el tamaño de imagen"
 
 #: ../src/iop/vignette.c:1194
 msgid "width-to-height ratio"
-msgstr "proporción ancho/alto"
+msgstr "Factor de proporción entre la altura y la anchura"
```
Comment (rgo) [inline po/es.po]: I think, original translation was more clear and **concise**.

File: po/es.po [hunk 9]
```diff
@@ -11398,15 +11400,15 @@ msgstr ""
...
 
 #: ../src/iop/vignette.c:1195
 msgid "add some level of random noise to prevent banding"
-msgstr "agregar algun nivel de ruido aleatorio para prevenir bandas"
+msgstr "Añadir cierto nivel aleatorio de ruido para prevenir bandas"
```
Comment (rgo) [inline po/es.po]: Current translated sentence means "add some random level of noise...".
I guess, it's more right to say "Añadir cierto nivel de **ruido aleatorio** para prevenir bandas"

File: po/es.po [hunk 10]
```diff
@@ -11467,16 +11469,16 @@ msgid ""
...
 "$(WATERMARK_COLOR)"
 msgstr ""
-"color de texto, etiqueta:\n"
+"Color de texto, etiqueta:\n"
 "$(WATERMARK_COLOR)"
 
 #: ../src/iop/watermark.c:1386
 msgid "select watermark color"
-msgstr "seleccionar color de la marca de agua"
+msgstr "Escoja el color de la marca de agua"
```
Comment (rgo) [inline po/es.po]: I would say "**Elige** el color..." to maintain the same wording everywhere.

File: po/es.po [hunk 11]
```diff
@@ -11554,15 +11556,15 @@ msgstr "desplazamiento y"
 #. Let's add some tooltips and hook up some signals...
 #: ../src/iop/watermark.c:1479
 msgid "the opacity of the watermark"
-msgstr "opacidad de la marca de agua"
+msgstr "Transparencia de la marca de agua"
```
Comment (rgo) [inline po/es.po]: A minor one, why not "Opacidad"?

I think in [photograpy terms we're looking for opacity](https://helpx.adobe.com/es/photoshop-elements/key-concepts/transparency-opacity.html).

File: po/es.po [hunk 12]
```diff
@@ -15159,27 +15161,27 @@ msgstr "cambiar el orden de las imágenes"
...
 
 #: ../src/views/lighttable.c:4637
 msgid "pan inside all the images"
-msgstr "Desplazarse dentro de todas las imágenes"
+msgstr "desplazar a la vez todas las imágenes"
 
 #: ../src/views/lighttable.c:4643
 msgid "zoom current image"
-msgstr "ampliar la imagen actual"
+msgstr "ampliar la imagen designada"
```
Comment (rgo) [inline po/es.po]: I would change "designada" by "actual".

File: po/es.po [hunk 13]
```diff
@@ -15159,27 +15161,27 @@ msgstr "cambiar el orden de las imágenes"
...
 
 #: ../src/views/lighttable.c:4649
 msgid "pan inside current image"
-msgstr "desplazarse dentro de la imagen actual"
+msgstr "desplazarse por la imagen designada"
```
Comment (rgo) [inline po/es.po]: Same as before, "designada" is not commonly used for this meaning, I prefer "actual".

File: po/es.po [hunk 14]
```diff
@@ -15260,39 +15262,39 @@ msgstr "No hay ninguna cámara con soporte de captura disponible para su uso..."
...
+msgstr "Clic Medio"
 
 #: ../src/views/view.c:2152
 msgid "Scroll"
 msgstr "Desplazamiento"
 
 #: ../src/views/view.c:2155
 msgid "Left double-click"
-msgstr "Doble clic izquierdo"
+msgstr "Pulsación Doble"
```
Comment (rgo) [inline po/es.po]: I would translate it as "Doble clic" (https://es.wikipedia.org/wiki/Doble_clic).
It's uncommon to say "pulsación doble".

File: po/es.po [hunk 15]
```diff
@@ -15260,39 +15262,39 @@ msgstr "No hay ninguna cámara con soporte de captura disponible para su uso..."
...
 
 #: ../src/views/view.c:2158
 msgid "Right double-click"
-msgstr "Doble clic derecho"
+msgstr "Doble Clic Secundario"
```
Comment (rgo) [inline po/es.po]: Case is not right at all "Doble clic secundario".

File: po/es.po [hunk 16]
```diff
@@ -15260,39 +15262,39 @@ msgstr "No hay ninguna cámara con soporte de captura disponible para su uso..."
...
 
 #: ../src/views/view.c:2161
 msgid "Drag and drop"
 msgstr "Arrastrar y Soltar"
 
 #: ../src/views/view.c:2164
 msgid "Left click+Drag"
-msgstr "Clic izquierdo + arrastrar"
+msgstr "Clic+Arrastre"
```
Comment (rgo) [inline po/es.po]: I would leave it in infinitive form, with capital letter only at the beginning and white spaces: "Clic + arrastrar"

File: po/es.po [hunk 17]
```diff
@@ -15260,39 +15262,39 @@ msgstr "No hay ninguna cámara con soporte de captura disponible para su uso..."
...
 
 #: ../src/views/view.c:2167
 msgid "Right click+Drag"
-msgstr "Clic derecho + arrastrar"
+msgstr "Clic Secundario + Arrastre"
```
Comment (rgo) [inline po/es.po]: Same as before: "Clic secundario + arrastrar"

File: po/es.po [hunk 18]
```diff
@@ -6630,7 +6630,7 @@ msgstr "ambos"
 
 #: ../src/iop/clipping.c:1989
 msgid "mirror image horizontally and/or vertically"
-msgstr "reflejar la imagen horizontal o verticalmente"
+msgstr "Espejar la imagen horizontal o verticalmente"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: "Espejar" is more accurate. "Voltear" can be in any direction.

File: po/es.po [hunk 19]
```diff
@@ -6725,15 +6725,15 @@ msgstr "3:1, panorámica"
...
 
 #: ../src/iop/clipping.c:2128
 msgid ""
 "set the aspect ratio\n"
 "the list is sorted: from most square to least square"
 msgstr ""
-"ajusta la relación de aspecto\n"
-"la lista está ordenada: desde el mas cuadrado hasta el menos cuadrado"
+"Fija la relación de aspecto\n"
+"La lista está ordenada desde la más cuadrada hasta la más rectangular"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: "desde *la más*" refers to aspect ratio, so the concordance.

File: po/es.po (hunk 13 above)
Comment (EdgarLux) [reply po/es.po]: "actual" loses its meaning in this context, as well as being vague. That's why the other word.

File: po/es.po (hunk 14 above)
Comment (EdgarLux) [reply po/es.po]: As in other cases I don't just use a data source. It is also congruent with other related messages.

File: po/es.po (hunk 16 above)
Comment (EdgarLux) [reply po/es.po]: It is more usual to have the key combinations without spaces unless there are compound words. And have the initial letter capitalized to specify that it is part of the key combination.
imagen

File: po/es.po (hunk 17 above)
Comment (EdgarLux) [reply po/es.po]: Same as before explanation

File: po/es.po (hunk 15 above)
Comment (EdgarLux) [reply po/es.po]: As in the following explanation...

File: po/es.po [hunk 20]
```diff
@@ -6708,7 +6708,7 @@ msgstr "2:1, univisium"
 
 #: ../src/iop/clipping.c:2034
 msgid "cinemascope"
-msgstr "cinemascope"
+msgstr "cinemascopio"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: As in some other cases, I chose the more common word. ( And only as additional information, there it is shown as a mark )

File: po/es.po [hunk 21]
```diff
@@ -7579,12 +7579,12 @@ msgstr "fuerza"
 
 #: ../src/iop/colorzones.c:2354
 msgid "choose between a smoother or stronger effect"
-msgstr "Elija entre un efecto más suave o más marcado."
+msgstr "Elegir entre un efecto más suave o más marcado."
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Rompe con el estilo de traducción del resto de mensajes. Remember that transliteration is not the best thing in translation

File: po/es.po [hunk 22]
```diff
@@ -7707,7 +7708,7 @@ msgstr "atravesar (monocromo) (experimental)"
 # Demosaicing -> Algoritmo de interpolación cromática.
 #: ../src/iop/demosaic.c:5130 ../src/iop/demosaic.c:5140
 msgid "demosaicing raw data method"
-msgstr "método de interpolación cromática RAW"
+msgstr "Método de interpolación cromática para imágenes en bruto"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Depending on the context I put one or the other, plus it gives more information to the newbies. I did it according to books, articles, manuals, etc.

File: po/es.po [hunk 23]
```diff
@@ -7891,8 +7892,8 @@ msgid ""
 "should be disabled if an earlier instance\n"
 "has been used with a color blending mode."
 msgstr ""
-"Adapta la reducción de ruido de acuerdo con los\n"
-"coeficientes de balance de blancos.\n"
+"Se adapta la reducción de ruido de acuerdo con los\n"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Nonsense "adapta". Who or what "adapta"? The impersonal form is best for this case.

File: po/es.po [hunk 24]
```diff
@@ -7991,14 +7992,14 @@ msgstr "ondículas automático"
 
 #: ../src/iop/denoiseprofile.c:4083
 msgid "profile used for variance stabilization"
-msgstr "perfil utilizado para estabilizar la varianza"
+msgstr "Perfil utilizado para estabilizar la varianza"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Se explica el proceso, por eso no es una traducción literal.

File: po/es.po [hunk 25]
```diff
@@ -8098,51 +8099,51 @@ msgstr "aleatorio"
...
 
 #: ../src/iop/dither.c:802
 msgid "floyd-steinberg auto"
-msgstr "algoritmo de floyd-steinberg automático"
+msgstr "Floyd-Steinberg automático"
 
 #: ../src/iop/dither.c:807
 msgid "radius for blurring step"
-msgstr "radio de desenfoque"
+msgstr "Radio para un paso de desenfoque"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: It's just that it goes by steps, that's why the correction. Check it in the module.

File: po/es.po [hunk 26]
```diff
@@ -8195,7 +8196,7 @@ msgstr "nivel de objetivo"
 
 #: ../src/iop/exposure.c:227
 msgid "magic lantern defaults"
-msgstr "propiedades de la linterna mágica"
+msgstr "parámetros de la linterna mágica"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: En este caso, eso sospecho. Sin embargo, no he podido confirmarlo al 100%. Y ya que la linterna mágica tiene otras referencias, hasta no verificarlo, opté por dejarla así por el momento.

File: po/es.po [hunk 27]
```diff
@@ -8219,9 +8220,9 @@ msgid ""
 "if poorly set, it will clip near-black colors out of gamut\n"
 "by pushing RGB values into negatives."
 msgstr ""
-"ajusta el nivel de negro para liberar valores RGB negativos.\n"
-"nunca debe usarlo para agregar más densidad en los negros. \n"
-"si está mal configurado, recortará los colores casi negros de la gama \\ n\n"
+"Ajusta el nivel de negro para retirar valores RGB de los negativos.\n"
+"No debe usarse para agregar más densidad en los negros. \n"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: They never and always tend to have another nuance in Spanish. En este caso, al ponerla, indicaría algo muy exagerado.

File: po/es.po (hunk 1 above)
Comment (EdgarLux) [reply po/es.po]: For the action, it is the corresponding verb.

File: po/es.po (hunk 2 above)
Comment (EdgarLux) [reply po/es.po]: Same as explanation above.

File: po/es.po (hunk 3 above)
Comment (EdgarLux) [reply po/es.po]: The score changes according to the language.

File: po/es.po (hunk 5 above)
Comment (EdgarLux) [reply po/es.po]: But that's what describes what the module does.

File: po/es.po (hunk 6 above)
Comment (EdgarLux) [reply po/es.po]: This was not a very simple translation. You have to use the module and then it makes perfect sense.

File: po/es.po (hunk 7 above)
Comment (EdgarLux) [reply po/es.po]: You have to use the module and then it makes perfect sense.

File: po/es.po (hunk 8 above)
Comment (EdgarLux) [reply po/es.po]: Not really, or I would have left it as it was. You have to use the modules.

File: po/es.po (hunk 11 above)
Comment (EdgarLux) [reply po/es.po]: Ese cambio lo hice por una razón que ahorita no tengo a la mano. Jajaja, pero lo reviso.

File: po/es.po (hunk 4 above)
Comment (EdgarLux) [reply po/es.po]: Same answer ;-P

File: po/es.po [hunk 28]
```diff
@@ -10529,35 +10530,35 @@ msgstr "reconstrucción de luces"
...
 
 #: ../src/iop/shadhi.c:868
 msgid "shift white point"
-msgstr "mover punto blanco"
+msgstr "Mover punto blanco"
 
 #: ../src/iop/shadhi.c:870
 msgid "filter to use for softening. bilateral avoids halos"
-msgstr "filtro de suavizado. bilateral evita los halos"
+msgstr "Filtro de suavizado. El bilateral evita los halos"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: By grammatical substitution

File: po/es.po [hunk 29]
```diff
@@ -8219,9 +8220,9 @@ msgid ""
 "if poorly set, it will clip near-black colors out of gamut\n"
 "by pushing RGB values into negatives."
 msgstr ""
-"ajusta el nivel de negro para liberar valores RGB negativos.\n"
-"nunca debe usarlo para agregar más densidad en los negros. \n"
-"si está mal configurado, recortará los colores casi negros de la gama \\ n\n"
+"Ajusta el nivel de negro para retirar valores RGB de los negativos.\n"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: In this case, there are negative values and positive values. It helps a lot to use the module.

File: po/es.po [hunk 30]
```diff
@@ -7534,7 +7534,7 @@ msgstr "película blanco y negro"
 #: ../src/iop/colorzones.c:2086 ../src/iop/retouch.c:2238
 #: ../src/iop/toneequal.c:2017
 msgid "cannot display masks when the blending mask is displayed"
-msgstr "no se puede mostrar máscaras cuando se muestra la máscara de mezcla"
+msgstr "No se puede mostrar máscaras cuando se muestra la de fusión"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: A typo! Thanks!

File: po/es.po [hunk 31]
```diff
@@ -7579,12 +7579,12 @@ msgstr "fuerza"
...
 #: ../src/iop/colorzones.c:2354
 msgid "choose between a smoother or stronger effect"
-msgstr "Elija entre un efecto más suave o más marcado."
+msgstr "Elegir entre un efecto más suave o más marcado."
 
 #: ../src/iop/colorzones.c:2390 ../src/iop/rgbcurve.c:1624
 #: ../src/iop/tonecurve.c:1405
 msgid "interpolation method"
-msgstr "método de interpolación"
+msgstr "interpolación"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: La etiqueta es muy larga y se sobrepone a las opciones haciéndolas ilegibles. That's the reason for the change.

File: po/es.po [hunk 32]
```diff
@@ -7792,15 +7793,15 @@ msgstr "promedio total y local"
...
 
 #: ../src/iop/demosaic.c:5178
 msgid ""
 "demosaicing\n"
 "only needed for raw images."
 msgstr ""
-"interpolación cromática\n"
-"solo es necesario en imágenes raw."
+"La interpolación cromática solo es\n"
+"necesaria para imágenes en bruto (RAW)."
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Like the other explanation

File: po/es.po [hunk 33]
```diff
@@ -8042,14 +8043,14 @@ msgid ""
...
-"Aumenta el peso del píxel central del parche\n"
+"Aumenta el peso del pixel central del parche\n"
 "al compararse los parches.\n"
 "Útil para recuperar detalles cuando el tamaño del parche\n"
 "es bastante grande."
 
 #: ../src/iop/denoiseprofile.c:4098
 msgid "finetune denoising strength"
-msgstr "fuerza de la corrección para reducir el ruido"
+msgstr "Fuerza de la corrección para reducir el ruido"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Eso sería una traducción muy literal. I'll make a change, though.

File: po/es.po [hunk 34]
```diff
@@ -8098,51 +8099,51 @@ msgstr "aleatorio"
...
+msgstr "atenuación"
 
 #: ../src/iop/equalizer.c:88
 msgid "legacy equalizer"
 msgstr "ecualizador (antiguo)"
 
 #: ../src/iop/equalizer.c:311
 msgid "sharpen (strong)"
-msgstr "enfoque (fuerte)"
+msgstr "enfoque (marcado)"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Of course, because all those meanings have strength in Spanish and many others.

File: po/es.po (hunk 10 above)
Comment (EdgarLux) [reply po/es.po]: You have to see where the message comes from in order to take into consideration the translation style.

File: po/es.po (hunk 9 above)
Comment (EdgarLux) [reply po/es.po]: Thanks!

File: po/es.po [hunk 35]
```diff
@@ -7648,26 +7649,26 @@ msgid ""
...
 
 #: ../src/iop/defringe.c:479
 msgid "radius for detecting fringe"
-msgstr "radio para detección de halos"
+msgstr "Radio para detección de halos"
 
 #: ../src/iop/defringe.c:480
 msgid "threshold for defringe, higher values mean less defringing"
-msgstr "umbral para reducir halos, mayores valores significan menor reducción"
+msgstr "Umbral para reducir halos. Mayores valores significan menor reducción"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Thanks!

File: po/es.po (hunk 18 above)
[the remark this answers is not in the document]
Comment (rgo) [reply po/es.po]: "Espejar" is not a common word at least in photography terms, in fact I have had to look for it in the dictionary (sounded wrong to me). Gimp, photoshop, lightroom,... all of them use "Voltear".

About the direction, you're specifying that after that "horizontally and/or vertically".

File: po/es.po (hunk 20 above)
[the remark this answers is not in the document]
Comment (rgo) [reply po/es.po]: Cinemascopio is not a common word (google returns 65.000 results).
Cinemascope is more used (google returns 3.800.000 results).

File: po/es.po (hunk 21 above)
[the remark this answers is not in the document]
Comment (rgo) [reply po/es.po]: I haven't been doing transliteration, it was a fault of context.

File: po/es.po (hunk 22 above)
[the remark this answers is not in the document]
Comment (rgo) [reply po/es.po]: "imágenes en bruto" doesn't mean anything, only confusion for newbies  because it is not used. Not relevant, but it was the first time I heard that.

If newbies are playing with a RAW processor, don't change the term :)

File: po/es.po [hunk 36]
```diff
@@ -7567,7 +7567,7 @@ msgstr "seleccionar por"
 
 #: ../src/iop/colorzones.c:2342
 msgid "choose selection criterion, will be the abscissa in the graph"
-msgstr "escoja criterio de selección, será la abscisa en el gráfico"
+msgstr "Elegir criterio de selección, que es la abscisa en el gráfico"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Again, it depends on the context.

File: po/es.po [hunk 37]
```diff
@@ -2591,8 +2591,8 @@ msgid ""
 "ctrl+click + drag to use the output image"
 msgstr ""
 "Determinar el rango basado en un área de la imagen\n"
-"Pulsar y arrastrar para usar la imagen de entrada\n"
-"Ctrl+Clic+Arrastrar para usar la imagen de salida"
+"Pulsar y Arrastrar para usar la imagen de entrada\n"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Yo're right!

**Comment by EdgarLux** [comment]
> Well done!
> 
> I don't understand the rational behind some global changes.
> There are some complex sentences that I've never could have done it better. Again, well done! :)
> 

Thanks!
> Just in case, from lines 15000 to 15181 initial capital letter is not used. I don't know where are used those translations and because of it, maybe, they don't need it.

Yes, indeed. That is the importance of seeing the correspondence of the message in the place of the interface. Generally speaking, it improves comprehension and readability.

File: po/es.po (hunk 20 above)
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: I mean in Spanish. Y el cómo es conocido. No por la frecuencia de palabras.

File: po/es.po (hunk 18 above)
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: 3. prnl. Reflejarse, reproducirse como la imagen en un espejo.

[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: No nada más es el hecho de voltearla, es el hecho de ponerla como en espejo.

**Comment by EdgarLux** [comment]
@TurboGit The changes have been made.

**Comment by TurboGit** [comment]
Thanks !

File: po/es.po (hunk 20 above)
[the remark this answers is not in the document]
Comment (rgo) [reply po/es.po]: In Spanish is cinemascope because it comes from a trademark, and RAE confirm that.

File: po/es.po (hunk 16 above)
Comment (rgo) [reply po/es.po]: Furthermore, adding spaces could probably be done with CSS know.

File: po/es.po (hunk 29 above)
[the remark this answers is not in the document]
Comment (rgo) [reply po/es.po]: Spanish translation don't say so:

"Ajusta el nivel de negro para **retirar valores RGB de los negativos.**\n"

File: po/es.po (hunk 28 above)
[the remark this answers is not in the document]
Comment (rgo) [reply po/es.po]: It remembers me when someone are taking notes, avoiding to duplicate terms in order to don't waste time. 🤷🏻‍♂️
